Api/Connection.php

```diff
@@ -10,7 +10,7 @@ class Connection {
          * @param Array $bodyArray
          * @return void
          */
-        public function post($url, $bodyArray = false)
+        public function post($url, $bodyArray = false, $hmac_auth = false)
```

Comment:
Sorry, my bad, `hmacAuth` so that it is aligned to the style of PHP

Reply:
Edited.